docs/en/query_language/functions/string_functions.md

```diff
@@ -115,13 +115,104 @@ SELECT format('{} {}', 'Hello', 'World')
 └───────────────────────────────────┘
 ```
 
-## concat(s1, s2, ...)
+## concat {#concat}
 
 Concatenates the strings listed in the arguments, without a separator.
 
-## concatAssumeInjective(s1, s2, ...)
+**Syntax** 
 
-Same as [concat](./string_functions.md#concat-s1-s2), the difference is that you need to ensure that concat(s1, s2, s3) -> s4 is injective, it will be used for optimization of GROUP BY
+```sql
+concat(s1, s2, ...)
+```
+Alias: `CONCAT`.
+
+**Parameters**
+
+- `si` – string or key. The key value can be any [data type](https://clickhouse.yandex/docs/en/data_types/).
```

Comment:
* internal links should point to markdown files instead of where they are published
* and supporting any data type sounds incorrect:
```
SELECT concat('aa', 1)
FORMAT TSV

Received exception from server (version 19.8.3):
Code: 386. DB::Exception: Received from localhost:9000, ::1. DB::Exception: There is no supertype for types String, UInt8 because some of them are String/FixedString and some of them are not.

0 rows in set. Elapsed: 0.001 sec.
```